ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -237,12 +252,15 @@ Item {
 
         model: messageStore.messagesModel
 
-        // Not Refactored Yet
-        //Component.onCompleted: scrollToBottom(true)
+        Component.onCompleted: chatLogView.scrollToBottom(true)
 
         delegate: MessageView {
             id: msgDelegate
 
+            // We wait to load all messages, because switching back to chats makes
```

Comment:
Wondering why is this required. The model-view-delegate in QML takes care of efficiently instantiating and destroying delegates based on the item count in the model and without the need to hide them.